nipype/interfaces/ants/utils.py

```diff
@@ -1,11 +1,456 @@
-# -*- coding: utf-8 -*-
-"""ANTS Apply Transforms interface
-"""
-
+"""ANTs' utilities."""
 import os
+from ...utils.imagemanip import copy_header as _copy_header
+from ..base import traits, isdefined, TraitedSpec, File, Str, InputMultiPath
+from .base import ANTSCommandInputSpec, ANTSCommand
+
+
+class _ImageMathInputSpec(ANTSCommandInputSpec):
```

Comment:
This convention was started in niworkflows but isn't generally used in nipype. Conventions that differ from file to file seems disorienting for contributors.

```suggestion
class ImageMathInputSpec(ANTSCommandInputSpec):
```

Reply:
I'd like to see NiPype slowly migrating to this convention. Would you be okay if the whole module included the change (adding aliases so that we don't break the work directories anytime soon)?

Reply:
Maybe let's have that conversation in a separate issue, rather than wedging it into this PR?

Reply:
Okay, sounds reasonable

Reply:
BTW - this is going to break fMRIPrep working directory.